docker-compose.yml

```diff
@@ -21,7 +25,7 @@ services:
     build:
       context: scripts
       dockerfile: Dockerfile-awslocal
-    entrypoint: awslocal
+    entrypoint: awslocal 
```

Comment:
**nitpick**

```suggestion
    entrypoint: awslocal
```

**thought:** In another repo we've had success with `yamllint` as a linter to ensure our files follow a consistent format. Unfortunately this tool doesn't come with an auto-fix function, so it can be a bit tedious. Maybe something to consider for the future, but not super urgent (or super high value TBH).